x-pack/plugins/ml/public/application/components/links/ml_href.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { ReactChild } from 'react';
+import { useMlKibana, useNavigateToPath } from '../../contexts/kibana';
+
+function isModifiedEvent(event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) {
+  return !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey);
+}
+
+export const MlHref = ({
+  href,
+  target,
+  onClick,
+  children,
+  ...rest
+}: {
+  href: string;
+  target?: string;
+  onClick?: Function;
+  children?: ReactChild;
+}) => {
+  const {
+    services: {
+      http: { basePath },
+    },
+  } = useMlKibana();
+
+  const navigateToPath = useNavigateToPath();
+  return (
+    <a
```

Comment:
question: have you tried using [EuiLink](https://elastic.github.io/eui/#/navigation/link) instead?

Reply:
EuiLink will not accept both `href` and `onClick`.

Reply:
strange, they have an example of combining them 
```typescript 
<EuiLink
        href="https://github.com/elastic/eui"
        onClick={(e) => {
          if (!window.confirm('Are you sure you want to see the eui repo?')) {
            e.preventDefault();
          }
        }}>
        link
      </EuiLink>
```

Reply:
Very strange... this is what I see whenever I use both. Maybe the eslint rule is outdated? I will reach out to Eui team. 
![Uploading Screen Shot 2020-10-16 at 12.43.32 PM.png…]()